scripts/i18n-utils/i18n-utils.js

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env node
+import fs from "fs";
+import path from "path";
+
+const srcFile = process.argv[2];
+const langFolder = process.argv[3];
+const baseLangFile = `${langFolder}/en.json`;
```

Comment:
I think all of these hard-coded/narrow assumptions are fine for now and we can pick it back up in our i18n followup effort.